### PR TITLE
Update gradle version

### PR DESCRIPTION
**Pull request by opacam**
To solve the ci errors when trying to download the corresponding build tools which seems to not exist anymore...

Resolves: #1506

### CODE REVIEW

**Comment by JonasT** [comment]
I did an additional build of the `testapp_keyboard` with Python 3 non-crystax / API 28 + NDK 21 / openjdk8 (p4aspaces' `p4a-py3-api28ndk21` env) because you know, I don't have much better things to do... and that one also worked. Haven't tested the `.apk` though, but the build runs through fine including the gradle step.

**Comment by opacam** [comment]
I did test an apk at run time and it seems to work as expected (only python2 tested but it should work for python 3 because I tested with the changes made in pr #1460, so both versions of python shares the same build system). The nature of the failure makes me think that all the ci builds made after the removed build tools will fail, unless restored the build tools...I don't know if we can rely on this...

The only visible difference (in functionality) I find is that the resulting apk's path has been changed into a sub folder (debug or release).

I must say that with this pr the most of the gradle dependencies will be downloaded from google...this will make the build system more reliable? hope that yes...

**Comment by inclement** [comment]
Thanks!

File: pythonforandroid/toolchain.py [hunk 1]
```diff
@@ -847,7 +847,9 @@ def apk(self, args):
 
                 # gradle output apks somewhere else
                 # and don't have version in file
-                apk_dir = join(dist.dist_dir, "build", "outputs", "apk")
+                apk_dir = join(dist.dist_dir,
+                               "build", "outputs", "apk",
+                               args.build_mode)
```
Comment (tito) [inline pythonforandroid/toolchain.py]: Just find out that this line breaks buildozer APK searchs. :[

Comment (tito) [reply pythonforandroid/toolchain.py]: Buildozer was searching in the apk directory, not a subdirectory.

Comment (opacam) [reply pythonforandroid/toolchain.py]: Ei @tito, it seems that @AndreMiras already fixed that in buildozer's master branch, see:

kivy/buildozer#766

**Pull request merged**